.gitlab-ci.yml

```diff
@@ -1,113 +1,123 @@
 stages:
-    - build
     - test
+    - package
     - deploy
 
-# Not executed, parent job definition for Maven builds
-.maven-build:
-    image: maven:3.5.4-jdk-8
-    stage: build
-    variables:
-        ARTIFACT_PATH: "eva-server/target/eva-$ENVIRONMENT_NAME.war"
-        _JAVA_OPTIONS: "-Djdk.net.URLClassPath.disableClassPathURLCheck=true"
-    script:
-        - "mvn package --settings .gitlab.settings.xml -P $MAVEN_PROFILE"
-        - "cp eva-server/target/eva-*.war $ARTIFACT_PATH"
-    artifacts:
-        paths:
-            - "$ARTIFACT_PATH"
-        expire_in: 2 weeks
+variables:
+    WS_ARTIFACT_PATH: eva-server/target/eva-$ENVIRONMENT_NAME.war
+    MAVEN_SETTINGS: maven-settings.xml
```

Comment:
```suggestion
    MAVEN_SETTINGS_XML: maven-settings.xml
```

Reply:
I actually prefer `MAVEN_SETTINGS` plus it will be consistent with what we have in the `eva-accession` pipeline